src/Reference/Task8.java

```diff
@@ -0,0 +1,44 @@
+package Reference;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Task8 {
+
+    public static void main(String[] args) {
+
+        //forexample
+//        String string = "ae 02 eo oo ao aa aeo sdf dkfm fkfkfk og Hello there this is a simple text for this awesome code 1234567";
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter a fraze: ");
+        String string = reader.nextLine();
+
+        String[] split = string.split(" ");
+        Pattern pattern = Pattern.compile("[aeo]");
```

Comment:
could you add a solution also for finding all letters aeo in a word, and also words with ae, ao, eo, that will be cool

Reply:
Pattern pattern = Pattern.compile("[aeoAEO]");